besu/src/main/java/org/hyperledger/besu/cli/custom/CorsAllowedOriginsProperty.java

```diff
@@ -85,14 +82,6 @@ public boolean addAll(final Collection<? extends String> collection) {
       if (domains.size() > 1) {
         throw new IllegalArgumentException("Values '*' or 'all' can't be used with other domains");
       }
-    } else {
```

Comment:
why remove this?

Reply:
Because the origin would not be treated as a regex anymore. Instead it would be compared as a string literal as in `addOrigin` in vertx 4.5.0.